cli/tests/test_new_command.rs

```diff
@@ -415,7 +438,8 @@ fn setup_before_insertion(test_env: &TestEnvironment, repo_path: &Path) {
     test_env.jj_cmd_ok(repo_path, &["branch", "create", "D"]);
     test_env.jj_cmd_ok(repo_path, &["new", "-m", "E", "root()"]);
     test_env.jj_cmd_ok(repo_path, &["branch", "create", "E"]);
-    test_env.jj_cmd_ok(repo_path, &["new", "-m", "F", "D", "E"]);
+    // Any number of -r-s is ignored
```

Comment:
nit: I find "-r-s" slightly confusing because it sounds a bit like "-r and -s". How about  "\`-r\`s"?